public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -305,11 +300,8 @@ export const ReceiversTable = ({ config, alertManagerName }: Props) => {
   const [receiverToDelete, setReceiverToDelete] = useState<string>();
   const [showCannotDeleteReceiverModal, setShowCannotDeleteReceiverModal] = useState(false);
 
-  const isGrafanaAM = alertManagerName === GRAFANA_RULES_SOURCE_NAME;
-  const showExport =
-    isGrafanaAM &&
-    (contextSrv.hasPermission(permissions.provisioning.read) ||
-      contextSrv.hasPermission(permissions.provisioning.readSecrets));
+  const [supportsExport, allowedToExport] = useAlertmanagerAbility(AlertmanagerAction.ExportContactPoint);
+  const showExport = supportsExport && allowedToExport;
```

Comment:
Theoretically yes, you could create a role that has the permissions for it but it would be unsupported when viewing the contact points for an external vanilla Alertmanager.